specreduce/extract.py

```diff
@@ -164,22 +164,57 @@ class BoxcarExtract(SpecreduceOperation):
     def spectrum(self):
         return self.__call__()
 
+    def _parse_image(self):
+        """
+        Convert all accepted image types to a consistently formatted Spectrum1D.
+        """
```

Comment:
What do you think of adding it as a class method in `SpecreduceOperation`? All the classes that need it inherit from there. `HorneExtract` needs a more stringent parser, but I could override the inherited `_parse_image()` by redefining it in `HorneExtract`.

Reply:
> What do you think of adding it as a class method in SpecreduceOperation?

I think that's a good idea if everything that inherits from `SpecreduceOperation` would make use of it... otherwise you could have a base class between that inherits from `SpecreduceOperation` and adds image functionality.  And I think even if you want to switch to the property idea, that could easily be migrated within that base class.

Reply:
I forgot that only the extraction operations inherit from `SpecreduceOperation`, so I tried putting `_parse_image` in an mixin class used as an ancestor for any class that needs its functionality instead. More details in the summary I posted below.